Sources/Commands/APIDigester.swift

```diff
@@ -199,14 +199,18 @@ public struct SwiftAPIDigester {
     public func compareAPIToBaseline(
         at baselinePath: AbsolutePath,
         for module: String,
-        buildPlan: BuildPlan
+        buildPlan: BuildPlan,
+        breakageAllowlistPath: AbsolutePath?
```

Comment:
For better readability,
```suggestion
        except breakageAllowlistPath: AbsolutePath?
```